EANvalidator/src/main/java/org/fundacionjala/CodingWar/EANValidator.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.CodingWar;
+
+/**
+ * Created by ErickaViraca on 8/26/2016.
+ */
+public class EANValidator {
+
+    /**
+     * Validates the EAN string
+     *
+     * @param EAN, string of value
+     * @return --> true or false
+     */
+    public static boolean validate(String EAN) {
+        int totalSum = additionResults(EAN);
+        final int divisor = 10;
+        int checksum = totalSum % divisor == 0 ? 0 : divisor - (totalSum % divisor);
+        return verifyChecksum(checksum, Integer.parseInt(String.valueOf(EAN.charAt(EAN.length() - 1))));
+    }
+
+    /**
+     * Adds the values of the final result for oods positions and pairs positions
+     *
+     * @param EAN, string of numbers
+     * @return -->  totalOddsPositions + totalPairsPositions
+     */
+    private static int additionResults(String EAN) {
```

Comment:
@ErickaViraca 
You can still improve this method.
because it is not necessary to have two variables totalOddsPositions and totalPairsPositions. it is just necessary one variable